app/views/posts/index.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>All Posts</h1>
-<p>This is a placeholder for posts index content.</p>
+<ul class="space-y-4">
+  <% @posts.each do |post| %>
+    <li class="bg-white shadow-md p-4 rounded-lg">
+      <h2 class="text-2xl font-semibold mb-2">
+        <%= link_to post.title, post, class: "text-blue-500 hover:underline" %>
+      </h2>
+      <p class="text-gray-700 mb-2"><%= post.text %></p>
+      <p class="text-gray-600">Author: <%= post.author.name %></p>
+      <p class="text-gray-600">Comments: <%= post.comments_counter %></p>
+      <p class="text-gray-600">Likes: <%= post.likes_counter %></p>
+    </li>
+  <% end %>
```

Comment:
- [ ] @aradradev Great work listing all the user's posts, however as shown in the initial design below each post is supposed to display the five most recent comments of the post, kindly consider adding it.
![image](https://github.com/aradradev/Blog-app/assets/61408860/b2d76c3d-9466-49ff-8623-a25cbb8981a7)